app/src/main/res/values-zh-rTW/androidcomponents_strings.xml

```diff
@@ -35,59 +35,59 @@
 		<li>若您以前可以與該伺服器正常連線，那麼這個錯誤可能只是暫時的，請稍候再試試看。</li>
 	</ul>
 	]]></string>
-	
+
 	<!-- The document title and heading of the error page shown when the user's network connection is interrupted while connecting to a website. -->
 	<string name="mozac_browser_errorpages_net_interrupt_title">連線中斷</string>
     <!-- The error message shown when the user's network connection is interrupted while connecting to a website. -->
     <string name="mozac_browser_errorpages_net_interrupt_message"><![CDATA[
-	<p>瀏覽器已連線成功，但傳輸資料時連接被中斷。請再試一次。</p> 
-	<ul> 
-		<li>此網站可能暫時無法使用或忙碌中。請稍後再試一次。</li>
-		<li>若您無法載入任何頁面，請檢查您裝置的數據或 Wi-Fi 連線。</li>
+	<p>瀏覽器連線成功，但傳輸被中斷了；請稍候重試。</p>
+	<ul>
+		<li>網站可能太忙碌或暫時無法使用，請稍候再試一次。</li>
+		<li>若您無法載入任何頁面，請檢查您的數據或 Wi-Fi 連線。</li>
 	</ul>
 	]]></string>
-	
+
 	<!-- The document title and heading of the error page shown when a website takes too long to load. -->
-	<string name="mozac_browser_errorpages_net_timeout_title">連線已逾時</string>
+	<string name="mozac_browser_errorpages_net_timeout_title">網路連線逾時</string>
     <!-- The error message shown when a website took long to load. -->
     <string name="mozac_browser_errorpages_net_timeout_message"><![CDATA[
-	<p>您要存取的網站對連線無回應，瀏覽器因此暫停等待回應。</p> 
-	<ul> 
-		<li>請檢查伺服器是否流量過高或是暫時離線？請稍後再試。</li>
-		<li>您可以瀏覽其他網站嗎？請檢查電腦的網路連線。</li>
-		<li>您的電腦或網路是否受防火牆或 Proxy 保護？錯誤設定可干擾網路瀏覽。</li>
-		<li>問題是否依然存在？請諮詢您的網路管理員或互聯網供應商尋求協助。</li>
+	<p>指定的網站一直沒有回應，瀏覽器已停止等待。</p>
+	<ul>
+		<li>該網站可能暫時流量過高？請稍候再試。</li>
+		<li>無法瀏覽其它網站？請檢查網路連線。</li>
+		<li>電腦有開防火牆或 Proxy？請確定這些設定是否正確。</li>
```

Comment:
I noticed `Proxy` was moved and changed to `Proxy？`. this look ok, @Delphine?

Reply:
Yes that looks ok, they use English words for certain terms